demos/tensorflow/tensorflow_training/install_python_with_conda.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
+
+# Install python and dependencies
+[ -f Miniconda3-latest-Linux-x86_64.sh ] || wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+[ -d miniconda ] || bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $script_dir/miniconda
+$script_dir/miniconda/bin/conda create --prefix $script_dir/python-occlum -y  python=3.7 tensorflow==1.15.0
+
+# Download mnist dataset
+[ -f mnist.npz ] wget https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz
```

Comment:
Is it correct?
[ -f mnist.npz ] || wget https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz